.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We probably don't want this `.vscode/settings.json` file checked into the repo.

Reply:
Added to .gitignore